Bt codec fixes

juimonen opened the pull request:
These are changes needed for bt offload to work with cavs-rt5682 topologies.

kv2019i commented on tools/topology/topology2/platform/intel/bt-default.conf:
```diff
@@ -1,15 +1,18 @@
+	BT_PB_HOST_PIPELINE_ID	7
+	BT_PB_DAI_PIPELINE_ID	8
+	BT_PB_HOST_PIPELINE_SINK "copier.SSP.8.1"
+	BT_PB_DAI_PIPELINE_SRC "copier.host.7.1"
```
The commit message should probably mention these example values were also reversed. This worked with cavs-nocodec as it specified this route explicitly.

juimonen replied:
ok

kv2019i commented on tools/topology/topology2/platform/intel/bt-generic.conf:
```diff
@@ -192,12 +192,12 @@ Object.Pipeline {
 
 Object.Base {
-	route."5" {
+	route."500" {
```
@ranj063 Do we have any better best-known-method to allocate the route ids?

ranj063 replied:
@kv2019i not at this time but I have a pending PR to avoid this https://github.com/alsa-project/alsa-utils/pull/188. Then all included conf files can start from 1 for their route IDs and the compiler will make them unique

kv2019i reviewed (approved):
Thanks @juimonen , a few minor notes, but nothing to stop merge.

juimonen commented:
updated:
- reordered signed-off lines in commit message
- commented in commit message the route reversal
- added BT_PCM_NAME "Bluetooth" to align with topology1
- SSP0/SSP2 switch topology bt_index = 1 (not 8)

kv2019i commented:
Looks good! @udaymb can you approve when you are good with this as well?

lgirdwood commented:
@juimonen conflicts and still RFC now ?

juimonen commented:
> @juimonen conflicts and still RFC now ?

Rebased now. Maybe @udaymb could still approve, we tested the "basic" topology, but not sof-mtl-max98357a-rt5682-ssp2-ssp0 (in real device), where ssp indexes are switched around. Can be however fixed later if problems found....
